core-library/src/main/java/org/silverpeas/core/util/URLUtil.java

```diff
@@ -405,7 +406,8 @@ public static String getMinifiedWebResourceUrl(final String url) {
     Component(URL_COMPONENT, "/Component/"), Folder(URL_TOPIC, "/Topic/"), File(URL_FILE, "/File/"),
     Document(URL_DOCUMENT, "/Document/"), Version(URL_VERSION, "/Version/"),
     Survey(URL_SURVEY, "/Survey/"), Question(URL_QUESTION, "/Question/"),
-    ForumMessage(URL_MESSAGE, "/ForumsMessage/"), Media(URL_MEDIA, "/Media/");
+    ForumMessage(URL_MESSAGE, "/ForumsMessage/"), Media(URL_MEDIA, "/Media/"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)